.github/workflows/test.yml

```diff
@@ -43,6 +43,14 @@ jobs:
 
       - name: Run Test
         working-directory: test
-        run: go test -v ./...
+        run: go test -v -timeout 250m ./...
         env:
           IC_API_KEY: ${{ secrets.ACCESS_KEY }}
+      - uses: 8398a7/action-slack@v2
+        with:
+          status: ${{ job.status }}
```

Comment:
This comment related to metadata.yml file

please add metadata.yml file which captures information about the module

refer: https://github.com/terraform-ibm-modules/terraform-ibm-observability/blob/main/metadata.yaml